### PR TITLE
feat: GEOT-711: styles for different small elements

### PR DESCRIPTION
**Pull request by ra8ga**

### CODE REVIEW

File: packages/ui/PostSingle/PostResponse/index.tsx [hunk 1]
```diff
@@ -1,38 +1,62 @@
-import React, { FC } from 'react'
-import styles from './styles.scss'
+// @ts-nocheck
```
Comment (ra8ga) [inline packages/ui/PostSingle/PostResponse/index.tsx:1]: this needs to be removed before merging

File: packages/ui/PostSingle/PostResponse/index.tsx [hunk 2]
```diff
@@ -1,38 +1,62 @@
...
+  createdAt: string
+  html: string
   noImage: { [key: string]: IImage }
+  owner: IUser
   user: IAccountDetails | null
-}> = ({ additionalStyles, html, createdAt, owner, noImage, user }) => {
+}> = ({ additionalStyles, createdAt, html, noImage, owner, user }) => {
+  const [userPreview, setUserPreview] = useState(false)
+  // @todo this needs to be replaced with real data
+  const mockeduser = {
```
Comment (ra8ga) [inline packages/ui/PostSingle/PostResponse/index.tsx:28]: this needs to be removed before merging

Comment (mariusz-kabala) [reply packages/ui/PostSingle/PostResponse/index.tsx:28]: fix this shit

File: packages/app-andrew/src/components/group/Sidebar/index.tsx [hunk 3]
```diff
@@ -41,7 +41,8 @@ export const GroupSidebar: FC<{}> = () => {
   if (!state.group) {
     return null
   }
-
+  // console.log(!state.hasAdminRights)
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/components/group/Sidebar/index.tsx:44]: remove comments

File: packages/app-andrew/src/components/user/InviteToGroupButton/styles.scss [hunk 4]
```diff
@@ -31,14 +33,65 @@
...
+          left: 0;
+          top: 0;
+          width: 100%;
+          height: 100%;
+          @include shadowBoxWithDescription();
+        }
+
+        .header {
+          position: absolute;
+          z-index: 1;
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/components/user/InviteToGroupButton/styles.scss:68]: you have variables for z-index, no?

File: packages/app-andrew/src/pages/group/[...params].tsx [hunk 5]
```diff
@@ -297,130 +297,160 @@ const GroupPage: NextPage<GroupPageProps> = (props) => {
...
-                      jeszcze żadnego posta... Możesz być pierwszy!
-                    </h3>
-                    <PostCreate groupId={state.group?.id || ''} />
+              <div>
+                <div className={styles.tagHeader}>
+                  <div>
+                    <span
+                      className={styles.image}
+                      style={{
+                        backgroundImage: `url('/images/temp-images/login_bg.png')`,
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/group/[...params].tsx:306]: what about that image file? is it final? or you going to replace it?

File: packages/app-andrew/src/pages/group/[slug]/styles.scss [hunk 6]
```diff
@@ -16,14 +16,31 @@
...
     max-width: 700px;
     min-height: 800px;
     background: $theme1;
 
+    &:after {
+      content: '';
+      position: absolute;
+      top: -8px;
+      left: 0;
+      z-index: 0;
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/group/[slug]/styles.scss:38]: take z-index from variable

File: packages/app-andrew/src/pages/group/styles.scss [hunk 7]
```diff
@@ -113,6 +156,7 @@
           background: $theme9;
 
           > div {
+            z-index: 1;
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/group/styles.scss:159]: z-index issue

File: packages/ui/PostSingle/DeletePost/index.tsx [hunk 8]
```diff
@@ -3,7 +3,7 @@ import cx from 'classnames'
 import { useTranslation } from '@gtms/commons/i18n'
 import useKey from 'use-key-hook'
 // ui
-import { Button } from '../../Button'
+import { Button } from '@gtms/ui/Button'
```
Comment (mariusz-kabala) [inline packages/ui/PostSingle/DeletePost/index.tsx:6]: that's internal import, was good, change it back, also fix Modal import below.

File: packages/app-andrew/src/components/group/Sidebar/index.tsx (hunk 3 above)
Comment (ra8ga) [reply packages/app-andrew/src/components/group/Sidebar/index.tsx:44]: removed

File: packages/app-andrew/src/components/user/InviteToGroupButton/styles.scss (hunk 4 above)
Comment (ra8ga) [reply packages/app-andrew/src/components/user/InviteToGroupButton/styles.scss:68]: fox z index: 1 I dont use variable, should I ? I consider it as internal zindex

File: packages/app-andrew/src/pages/group/[slug]/styles.scss (hunk 6 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/group/[slug]/styles.scss:38]: this cannoot break nothing

File: packages/app-andrew/src/pages/group/styles.scss (hunk 7 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/group/styles.scss:159]: this cannot break nothing

File: packages/app-andrew/src/pages/group/[...params].tsx (hunk 5 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/group/[...params].tsx:306]: for now I would stay with static image - different one, but still static

File: packages/ui/PostSingle/DeletePost/index.tsx (hunk 8 above)
Comment (ra8ga) [reply packages/ui/PostSingle/DeletePost/index.tsx:6]: fixed

**Review by mariusz-kabala — APPROVED** [review]

**Pull request merged**